manticore/core/cpu/abstractcpu.py

```diff
@@ -605,7 +724,15 @@ def render_instruction(self):
 
     def render_register(self, reg_name):
         result = ""
-        value = self.read_register(reg_name)
+
+        self.will_read_register.disable()
```

Comment:
`render_register` is only called from `render_registers`. Move the disable/renable stuff there, otherwise we're disabling/reenabling all the signals a ton unnecessarily.